tox.ini

```diff
@@ -45,13 +45,9 @@ usedevelop = False
 # tests.
 extras = test
 commands =
+# `tahoe --version` is not ready for prime time yet under py36. May it be some day!
+         !py36: tahoe --version
          trial {env:TAHOE_LAFS_TRIAL_ARGS:--rterrors} {posargs:allmydata}
-         tahoe --version
-
-[testenv:py36]
-# On macOS, git inside of ratchet.sh needs $HOME.
-passenv = {[testenv]passenv} HOME
-commands = {toxinidir}/misc/python3/ratchet.sh
```

Comment:
My understanding here is that since the test invocation for py36 is now the same as for py27, no dedicated section is required - `tox -e py36` will run the commands from `testenv`, but in a virtualenv with python 3.6 installed in it?

Reply:
[Yes](https://tox.readthedocs.io/en/latest/config.html#tox-environments)! And then a [factor condition](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings) prevents `tahoe --version` from being run in the `py36` environment:

https://github.com/tahoe-lafs/tahoe-lafs/blob/ae87037e737267b1b49ee9a270927228fbae1b36/tox.ini#L49